.nvd/config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "nvd": {"suppression-file": ".nvd/suppression.xml"}
```

Comment:
@kelvinqian00 this is a good example of why we put config in the root, file paths in config files. This works (because the .nvd/ added), but is immediately more confusing because it knows nothing of the relative path in the dir. In this case everything works so let's just go with it.